EnergyPlus_oM/OrderAttribute.cs

```diff
@@ -26,11 +26,12 @@
 using System.Runtime.CompilerServices;
 using System.Text;
 using System.Threading.Tasks;
+using BH.oM.Base;
 
-namespace BH.oM.Reflection
+namespace BH.oM.Base.Attributes
 {
     [AttributeUsage(AttributeTargets.Property, Inherited = false, AllowMultiple = false)]
-    public class OrderAttribute : Attribute
+    public class OrderAttribute : Attribute, IObject
```

Comment:
WOnder if this needs verisoning support? Might not be in the list, as it was not an IObject before.

Also, not sure if the internal structure of this class makes sense? The constructor sets a private field, but then there is a public proeprty, that has 0 reference to the set field?

Not scope of this PR, but just highlighting as seen.

Reply:
I do agree @IsakNaslundBh but am going to leave for now as out of scope of the immediate problem. This toolkit is alpha only so versioning isn't needed, but @jamesramsden-bh could you review this at some point to fix or take the toolkit out of alpha?

I'm going to get @EKAdebo to review this PR using a test script to verify we're ok to merge this as part of the work for the `IObject` compliance.